backend/src/org/commcare/xml/DetailParser.java

```diff
@@ -223,10 +217,10 @@ public Detail parse() throws InvalidStructureException, IOException, XmlPullPars
                         }
                     } else if (name.equals("background")) {
                         if (nextTagInBlock("background")) {
+                            // parse but don't use
+                            // TODO PLM: is there a way to just skip over entire tag?
                             checkNode("text");
-                            //Get it if so
-                            background = new TextParser(parser).parse();
-                            builder.setBackground(background);
+                            new TextParser(parser).parse();
```

Comment:
Hey, there is actually a method in our internal parser to skip a tag body. Would be good to add a detail parse including such a tag to a regression to make sure the jump method actually works, though, since jumping over tags in stream parsing can get dicey. Grab me if you want to do so before we pull this, otherwise, I'm good to pull